src/renderer/scss/component/_shapeshift.scss

```diff
@@ -0,0 +1,63 @@
+.shapeshift__wrapper.shapeshift__initial-wrapper {
+  min-height: 311px;
+}
+
+.shapeshift__content {
+  .spinner {
+    margin-top: 75px;
+  }
+}
+
+.shapeshift__success {
+  color: var(--color-primary);
+}
+
+.shapeshift__error {
+  color: var(--color-error);
+}
+
+.shapeshift__error,
+.shapeshift__maxdeposit {
+  min-height: 25px;
+}
+
+.shapeshift__deposit-address-wrapper {
+  display: flex;
+  flex-direction: row;
+
+  * {
+    align-self: center;
+  }
+}
+
+.shapeshift__deposit-address {
+  max-width: 500px;
+  font-size: 22px;
+  word-wrap: break-word;
+}
+
+.shapeshift__qrcode {
+  // don't use a variable here. adds a white border for easier reading in dark mode
+  // needs to stay the same no matter what theme is present
+  background-color: #fff;
+  padding: 2px;
+  margin-left: 40px;
+}
+
+.shapeshift__received,
+.shapeshift__complete {
+  padding: 20px 0;
+}
+
+.shapeshift__actions {
+  padding-top: 20px;
```

Comment:
Most of these paddings should be `$spacing-vertical` or 1/2 `$spacing-vertical`.